src/ol_superset/Dockerfile

```
@@ -1,6 +1,7 @@
 FROM apachesuperset.docker.scarf.sh/apache/superset:${SUPERSET_TAG:-latest}
 COPY --link requirements.in requirements.txt
 COPY --link pythonpath /app/pythonpath
+COPY --link static /app/static
```

This completely overrides the static directory in the docker image, which isn't what we want. Instead we just want to copy the file into the target directory.
```suggestion
COPY static/assets/images/ol-data-platform-logo.svg /app/static/assets/images/ol-data-platform-logo.svg
```